core/src/actors/sessions_manager.rs

```diff
@@ -124,18 +121,19 @@ impl Actor for SessionsManager {
         debug!("Sessions Manager actor has been started!");
 
         // Send message to config manager and process its response
-        send_get_config_request(self, ctx, |s, _ctx, config| {
+        send_get_config_request(self, ctx, |act, ctx, config| {
```

Comment:
What does `act` stand for? Actix? If so, I'd rather use `actix`. Let's make our code as eloquent as possible!